pulsenotify/consumer.py

```diff
@@ -243,7 +243,10 @@ def log_data(self):
                 }
 
     def make_s3_key(self, run_id):
-        build_properties = self.definition['extra']['build_props']
+        # extra.build_props is for release promotion in releasetasks.
+        # payload.properties is for the in-tree one.
+        build_properties = self.definition['extra'].get('build_props') or \
+            self.definition['payload']['properties']
```

Comment:
I'm not sure if this is going to work. In releasetasks we explicitly set `extra.build_props` for all tasks we generate. In the in-tree world the tasks are all different. [This is](https://tools.taskcluster.net/groups/NnPn1IvtQqq9ur84LyqhWg/tasks/XtVMHbo6T6mYeh-q9pzuYQ/details) another task from the same graph, for example. The approach will work for BBB tasks, but not for all other tasks. 

Maybe we should stop copying the logs all together? We can find them from releasewarrior or the TC index, even though it's not as convenient as finding them in a  single bucket. @nthomas-mozilla, what you say?